pkg/fanal/image/daemon/podman.go

```diff
@@ -25,10 +25,15 @@ type podmanClient struct {
 	c http.Client
 }
 
-func newPodmanClient() (podmanClient, error) {
+func newPodmanClient(host string) (podmanClient, error) {
 	// Get Podman socket location
 	sockDir := os.Getenv("XDG_RUNTIME_DIR")
 	socket := filepath.Join(sockDir, "podman", "podman.sock")
+	if host != "" {
+		socket = host
+	} else if s := os.Getenv("PODMAN_HOST"); s != "" {
```

Comment:
`DOCKER_HOST` is natively supported by Docker.
https://docs.docker.com/engine/reference/commandline/cli/

But I didn't find PODMAN_HOST support in Podman. If Podman doesn't support it, there is no reason we do it.